spec/features/user_can_log_in_and_out_spec.rb

```diff
@@ -4,14 +4,14 @@
     scenario "logs into users account" do
       user = create(:user)
 
-      visit "/"
+      visit root_path
```

Comment:
I personally like path helpers more because if you change the routes but don't change the prefixes they should still work.  But that rarely happens.